json/Arles_Text.txt

```diff
@@ -6036,423 +6036,423 @@
 		"jp_name": "アーレスソード",
 		"tr_name": "Ares Sword",
 		"jp_text": "ぐうっ……！<%br>ぶ、ブレード……てめぇ……",
-		"tr_text": "",
+		"tr_text": "Gah...!<%br>B-Blade... You...",
 		"fileID": 2
 	},
 	{
 		"eventNo": "15",
 		"jp_name": "アーレスブレード",
 		"tr_name": "Ares Blade",
 		"jp_text": "大人しくしてなさい。元兄弟のよしみ……<%br>ここにいれば、アンタたちは助けてあげる。",
-		"tr_text": "",
+		"tr_text": "Be silent. If you care about your comrades...<%br>You can help them by staying here.",
 		"fileID": 2
 	},
 	{
 		"eventNo": "16",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "か、カイゼルさんは……キャアッ！",
-		"tr_text": "",
+		"tr_text": "K-Kaiser is... Yaah!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "17",
 		"jp_name": "ブルーノ",
 		"tr_name": "Bruno",
 		"jp_text": "奇襲、ね……<%br>ネストにこんな隠し玉があったとは。",
-		"tr_text": "",
+		"tr_text": "An ambush, huh...<%br>So this is what Nest had hidden up her sleeve.",
 		"fileID": 2
 	},
 	{
 		"eventNo": "18",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "リーダー、みんな！　見つけたぞ！<%br>爆撃はあそこにいるヤツだ！",
-		"tr_text": "",
+		"tr_text": "Leader, everyone! I found them!<%br>The shooter is that guy over there!",
 		"fileID": 2
 	},
 	{
 		"eventNo": "19",
 		"jp_name": "アネット",
 		"tr_name": "Annette",
 		"jp_text": "あれは……彼も、アーレスの……",
-		"tr_text": "",
+		"tr_text": "That man... He, too, is an Ares...",
```

Comment:
I think the language chosen is definitive that Launcher is male but I don't know if his design has been revealed.

Reply:
`That's... an Ares...`

Reply:
Works for me